.github/actions/openziti-tunnel-build-action/redhat-8/Dockerfile

```diff
@@ -44,7 +44,7 @@ ENV VCPKG_FORCE_SYSTEM_BINARIES=yes
 
 RUN cd /usr/local \
     && git clone https://github.com/microsoft/vcpkg \
-    && ./vcpkg/bootstrap-vcpkg.sh -disableMetrics
```

Comment:
the bootstrap steps was moved to the entrypoint script so that it would run as the effective user of the container instead of root